### ISSUE TITLE
GM can no longer select a player token to see what they can see.

### ISSUE THREAD

**Issue by JesterOC** [foundryvtt/pf2e#13062]
This appears to be fairly new behavior. I have not found a setting that might revert it back to the original system.

Currently as a GM if you select a player's token that have PC vision enabled you no longer are able to see what they can see. Previously I would use this to see what their limits of vision are when describing rooms, or determining valid targets.

**Comment by stwlam** [comment]
It sounds like you have GM vision turned on. Is the light on the left purple and fluorescent?

**Comment by JesterOC** [comment]
The light on the left is standard. 

I checked the configuration settings and party vision is off.
I tried turning it on then off again and it did not work. When I did turn it on the light on the left did not look different.
![image](https://github.com/foundryvtt/pf2e/assets/954708/80207bdb-3b25-4f8d-aff4-d69dc54f1a43)

**Comment by stwlam** [comment]
Try disabling modules.

**Comment by JesterOC** [comment]
Done, no change.

![image](https://github.com/foundryvtt/pf2e/assets/954708/502b46a0-fa19-4a38-bd42-fc5fccd1fa9f)
Though I did not restart Foundry
I will try that

**Comment by JesterOC** [comment]
No change on restart

**Comment by JesterOC** [comment]
Seeing something odd. This was happening in two saved games. When I jumped in to the other one. I notice that the light bulb looks different (CF bulb vs incandescent) and then I clicked that I saw that a gm vision icon (the CFL) bulb was there (it was not in the other world). I did notice that Ctrl-G was the key stroke needed to toggle it. And that seems to have fixed that world.

I jumped back to the one I was using to show you and toggled Ctrl-G and now it seems to work, even has the CLF bulb where it was not showing up before.

Strange but I will take it. Thanks for the help. Sorry for the bother.
![image](https://github.com/foundryvtt/pf2e/assets/954708/8f2f31e3-0db9-4f97-b852-80a527da228e)
